contents/customers/wittyworks.md

```diff
@@ -0,0 +1,54 @@
+---
+title: How Witty Works created open-source plugins with the PostHog community
+customer: Witty Works
+logo: ../images/customers/wittyworks/witty-logo.png
+featuredImage: ../images/customers/wittyworks/featured.png
+industries:
+    - SaaS, Browser extension
+users:
+    - Marketing
+    - Engineering
+toolsUsed:
+    - Dashboards
+    - Plugins
+---
+
+[Witty Works](https://www.witty.works/) is the creator of Witty, an AI-based assistant which guides users to become better, more inclusive writers. Since launching in 2018 it has been adopted by organizations such as Microsoft and Deloitte to help discourage unconscious bias in written communications. 
+
+Initially, the team built their own tool for gathering product data, writing a JSON file into the file system for every event observed. Eventually, this proved untenable and unscalable. 
```

Comment:
'untenable' while correct I guess feels quite vague/formal?

Reply:
```suggestion
Initially, the team built their own tool for gathering product data, writing a JSON file into the file system for every event observed. Eventually, this proved unscalable. 
```

Even worse, it's redundant in meaning. Thanks Charles!